changelogs/unreleased/4424-rajatvig-small.md

```diff
@@ -0,0 +1 @@
+Add grpc_stats filter for Envoy
```

Comment:
might want to also add a note that this will give you the ability to log grpc status codes in access logs as well

Reply:
Done